src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1258,41 +1235,63 @@ bistTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::pt
 }
 
 /*
- * TEST #11
- */
-void
-vcuKernelTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptree& _ptTest)
+* Helper function to perform an initial check of whether a test is supported on device
+*/
+bool
```

Comment:
Please refactor this PR do no longer user C style code function return variables.